### ISSUE TITLE
ZipkinSpec doesn't tear down zookeeper instance

### ISSUE THREAD

**Issue by franklinhu** [openzipkin/zipkin#22]
When running tests for zipkin server, the `ZipkinSpec` doesn't close the ZK instance that's started. This causes another run of the tests to fail since the ZK instance is still bound to the port.

To reproduce:

```
  $ bin/sbt
  > test // this may pass
  > ...
  > test // ZipkinSpec will fail
```

**Comment by franklinhu** [comment]
Also we may want to consider using this to stand up the ZK instance: http://twitter.github.com/commons/apidocs/com/twitter/common/zookeeper/testing/ZooKeeperTestServer.html

**Comment by franklinhu** [comment]
Fixed in #23
